genesis/bootstrappers.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package genesis
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	_ "embed"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/constants"
+	"github.com/ava-labs/avalanchego/utils/ips"
+	"github.com/ava-labs/avalanchego/utils/sampler"
+)
+
+var (
+	//go:embed bootstrappers.json
+	bootstrappersPerNetworkRawJSON []byte
+
+	bootstrappersPerNetwork map[string][]Bootstrapper
+)
+
+func init() {
+	err := json.NewDecoder(bytes.NewReader(bootstrappersPerNetworkRawJSON)).Decode(&bootstrappersPerNetwork)
+	if err != nil {
+		panic(fmt.Sprintf("failed to decode bootstrappers.json %v", err))
+	}
+}
+
+// Represents the relationship between the nodeID and the nodeIP.
+// Sometimes called "anchor" or "beacon" node.
+type Bootstrapper struct {
+	ID ids.NodeID `json:"id"`
+	IP ips.IPDesc `json:"ip"`
+}
+
+// SampleBootstrappers returns the some beacons this node should connect to
+func SampleBootstrappers(networkID uint32, count int) []Bootstrapper {
+	networkName, ok := constants.NetworkIDToNetworkName[networkID]
+	if !ok {
+		return nil
+	}
```

Comment:
Feel free to ignore - but this `if` could just be deleted (doesn't do anything)... If you think it's more readable as is then I'm fine with it -shrug-